### DISCUSSION TITLE
How to start up  backend of Skywaking(version 9.4.0 to version 9.7.0  )？  error log: this version of the Java Runtime only recognizes class file versions up to 52.0

### DISCUSSION THREAD

**Discussion by DeBruyne2020** [apache/skywalking#12045] [Q&A] [answered]
I have a question for help 🙏

When I start up skywalking Skywaking(version 9.7.0, version 9.4.0) in linux    ( do command:   sh start.sh ),    I find it fail to start and error log of oap.log are as follow: 

Error: A JNI error has occurred ,please check your installation and try again Exception in thread "main" java.lang.UnsupportedClassVersionError: org/apache/skywalking/oap/server/starter/OAPServerStartUp has beean compiled by a more recent version of the Java Runtime( class file version 55.0), this version of the Java RUNtime only recognizes class file versions up to 52.0

so how to solve the problem?,   how can we success to start up the skywalking backend?     
do we have to upgrade our java jave version to jdk11 ?
excep upgrade javaversion,  are there any other solution ?

help🙏

**Comment by wu-sheng** [comment, marked as answer]
This is an issue you could simply google.
JRE should be 11+, but you local seems to be JRE8.

**Comment by Superskyyy** [comment]
This question could easily be answered by our AI bot on the website.  https://skywalking.apache.org/docs/
To start up the backend of SkyWalking, you need to use the `bin/startup.sh` (or cmd) script which is located in the `/bin` folder of the SkyWalking's backend distribution package. This script starts up the backend and UI with their default settings. 

However, the error log you're seeing suggests that there's a compatibility issue with the version of Java Runtime you're using. SkyWalking requires JDK11 or JDK17. If you're using a version of Java that only recognizes class file versions up to 52.0, it means you're using Java 8. You need to upgrade your Java version to either JDK11 or JDK17 to resolve this issue.

Here are the steps to start up the backend:

1. Ensure you have JDK11 or JDK17 installed on your system.
2. Navigate to the `/bin` folder of the SkyWalking's backend distribution package.
3. Run the `bin/startup.sh` (or cmd) script to start up the backend and UI.

Please note that the main purpose of the quickstart is to help you obtain a basic configuration for previews/demos. Performance and long-term running are not among the purposes of the quickstart. For production/QA/tests environments, you should refer to the [Backend and UI deployment documents](https://skywalking.apache.org/docs/main/latest/en/setup/ui-setup).

Source: [SkyWalking Backend Setup](https://skywalking.apache.org/docs/main/latest/en/setup/backend/backend-setup#backend-setup)
